src/boost_histogram/_internal/storage.py

```diff
@@ -17,6 +17,8 @@ def __repr__(self):
 @set_family(MAIN_FAMILY)
 @set_module("boost_histogram.storage")
 class Int64(store.int64, Storage):
+    def __init__(self):
+        self._PyType = int()
```

Comment:
I think this is causing the segfault in the CI, as Int64 is not properly initializing store.int64 (but could be wrong). The fact the tests are segfaulting is slightly bothering me.